src/styles/index.css

```diff
@@ -1,9 +1,81 @@
-.next-lib-template_container {
-    background-color: black;
-    color: white;
-
-    &::after {
-        content: '';
-        margin-bottom: 50px;
-    }
+.next-button_button {
+    position: relative;
+    width: 100%;
+    display: inline-block;
+    padding: 1.5rem 0;
+    border: none;
+    background-color: #aaa;
+    color: #000;
+    cursor: pointer;
+    text-decoration: none;
+    text-align: center;
+}
+
+/* ==========================================================================
+   normalise.css
+   https://github.com/necolas/normalize.css/blob/master/normalize.css
+   ========================================================================== */
+
+/**
+ * 1. Change the font styles in all browsers.
+ * 2. Remove the margin in Firefox and Safari.
+ */
+
+ button.next-button_button {
```

Comment:
I understand the reason behind this, but this creates a specificity of `11`.  So I imagine we will have problems trying to override this properties with a single class (specificity `10`).

Maybe we should have 2 classes:
`next_button` and `next-button_button`

The first one will behave like the current `next-button_button`.

The  `next-button_button` will be for the button only. (We wouldn't pass it if it is a link)

Reply:
💯, good thinking!

To make it more explicit I reverted to having both `.next-button_link` and `.next-button_button` classes, and applied it accordingly!